test/e2e/common/security_context.go

```diff
@@ -69,24 +69,20 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		}
 
 		/*
-		  Release : v1.12
-		  Testname: Security Context: runAsUser (id:65534)
-		  Description: Container created with runAsUser option, passing an id (id:65534) uses that
-		  given id when running the container.
-		  This test is marked LinuxOnly since Windows does not support running as UID / GID.
+			Release : v1.14
+			Testname: Security Context, runAsUser (id:65534)
+			Description: Container is created with runAsUser option by passing an id (id:65534) to run as unpriviledged user. Pod MUST be in Succeeded phase.
 		*/
-		It("should run the container with uid 65534 [LinuxOnly] [NodeConformance]", func() {
+		framework.ConformanceIt("should run the container with uid 65534 [NodeConformance]", func() {
```

Comment:
Please do not remove the ``[LinuxOnly]`` label, test contains a SecurityContext, which is not working on Windows.

Reply:
Reverted to keep LinuxOnly tag.